packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -1051,21 +1062,45 @@ const New = decorate([
                         </FormGroup>
                       )}
                       {state.isDelta && (
-                        <FormGroup>
-                          <label htmlFor={state.inputPreferNbd}>
-                            <strong>{_('preferNbd')}</strong>{' '}
-                            <Tooltip content={_('preferNbdInformation')}>
-                              <Icon icon='info' />
-                            </Tooltip>
-                          </label>
-                          <Toggle
-                            className='pull-right'
-                            id={state.inputPreferNbd}
-                            name='preferNbd'
-                            value={preferNbd}
-                            onChange={effects.setPreferNbd}
-                          />
-                        </FormGroup>
+                        <div>
+                          <FormGroup>
+                            <label htmlFor={state.inputPreferNbd}>
+                              <strong>{_('preferNbd')}</strong>{' '}
+                              <Tooltip content={_('preferNbdInformation')}>
+                                <Icon icon='info' />
+                              </Tooltip>
+                            </label>
+                            <Toggle
+                              className='pull-right'
+                              id={state.inputPreferNbd}
+                              name='preferNbd'
+                              value={preferNbd}
+                              onChange={effects.setPreferNbd}
+                            />
+                          </FormGroup>
+                          <FormGroup>
+                            <label htmlFor={state.inputCbtDestroySnapshotData}>
+                              <strong>{_('cbtDestroySnapshotData')}</strong>{' '}
+                              <Tooltip
+                                content={_(
+                                  preferNbd && !state.snapshotMode
+                                    ? 'cbtDestroySnapshotDataInformation'
+                                    : 'cbtDestroySnapshotDataDisabledInformation'
+                                )}
```

Comment:
Move the "disabled tooltip" on the toogle.
```suggestion
                              <Tooltip content={_('cbtDestroySnapshotDataInformation')}>

```